apps/web/src/pages/ApplicantDashboardPage/components/QualifiedRecruitments/QualifiedRecruitments.tsx

```diff
@@ -0,0 +1,216 @@
+  activeRecruitments: Application[];
+  expiredRecruitments: Application[];
+}
+
+type AccordionItems = Array<"active" | "expired" | "">;
+
+const QualifiedRecruitments = ({
+  activeRecruitments,
+  expiredRecruitments,
+}: QualifiedRecruitmentsProps) => {
+  const intl = useIntl();
+
+  const [currentAccordionItems, setCurrentAccordionItems] =
+    React.useState<AccordionItems>([
+      activeRecruitments.length > 0 ? "active" : "",
+      expiredRecruitments.length > 0 ? "expired" : "",
+    ]);
+
+  return (
+    <section>
+      <div>
+        <Heading level="h2" data-h2-font-weight="base(400)">
+          {intl.formatMessage({
+            defaultMessage: "My qualified recruitments",
+            id: "wYe3P2",
+            description:
+              "Heading for my qualified recruitments section on the applicant dashboard.",
+          })}
+        </Heading>
+        <p data-h2-margin="base(x1, 0, 0, 0)">
+          {intl.formatMessage({
+            defaultMessage:
+              "Recruitments are pools of talent that Government of Canada managers can hire from. This section lists all recruitments you’re a part of, categorized into two groups:",
+            id: "qU2ffO",
+            description:
+              "Description for my qualified recruitments section on the applicant dashboard.",
+          })}
+        </p>
+        <ul
+          data-h2-margin="base(x1, 0, 0, 0)"
+          data-h2-padding="base(0, 0, x2, x1)"
+        >
+          <li>
+            <p>
+              {intl.formatMessage({
+                defaultMessage:
+                  "<strong>Active</strong> recruitments are processes that are currently being hired from. You can control whether your profile shows up when a manager requests talent from this group of people.",
+                id: "jdTHer",
+                description:
+                  "Active list item for my qualified recruitments section on the applicant dashboard.",
+              })}
+            </p>
+          </li>
+          <li>
+            <p>
+              {intl.formatMessage({
+                defaultMessage:
+                  "<strong>Expired</strong> recruitments are processes that have ended either because they’ve become stale or because a new, more up-to-date recruitment has replaced them. You will not show up in results related to expired recruitments.",
+                id: "qYjmRp",
+                description:
+                  "Expired list item for my qualified recruitments section on the applicant dashboard.",
+              })}
+            </p>
+          </li>
+        </ul>
+      </div>
+      <div>
+        <Accordion.Root
+          type="multiple"
+          mode="simple"
+          value={currentAccordionItems}
+          onValueChange={(newValue: AccordionItems) => {
+            setCurrentAccordionItems(newValue);
+          }}
+        >
+          {/* Active Recruitments */}
+          <Accordion.Item value="active">
+            <Accordion.Trigger>
```

Comment:
We should slap a `headerAs="h3"` on here since it is technically heading a sub section under "My qualified recruitments"

Reply:
Fixed by [997d65f](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5988/commits/997d65f03087c750485b174c0aeaa08aeec6b1e0)